pyulog/core.py

```diff
@@ -73,14 +73,30 @@ def get_field_size(type_str):
     _unpack_ushort = struct.Struct('<H').unpack
     _unpack_uint64 = struct.Struct('<Q').unpack
 
+    # when set to True disables string parsing exceptions
+    _disable_str_exceptions = False
+
+    @staticmethod
```

Comment:
Why is this static?

Reply:
You mean instead of a member function of ULog? The wrapper needs to either be global or static to be accessible in the 'sub' classes, or is there another way classes like MessageLogging can access ULog's member functions without access to ULog object itself?

Reply:
Ah right, the subclasses. Ok, passing the ulog object is not nice either.